uber/configspec.ini

```diff
@@ -49,6 +49,15 @@ use_checkin_barcode = boolean(default=False)
 # registering, False to disallow promo codes.
 badge_promo_codes_enabled = boolean(default=False)
 
+# This controls the window after "prereg_open" during which the "Request
+# Hotel Booking Info" step is including in the preregistration workflow.
+# The duration is expressed as a number of seconds. For example: to keep
+# "Hotel Info" open for a 24 hour window after prereg launches set this
+# value to 86400 (24 hours * 60 minutes * 60 seconds).
```

Comment:
Hmm. I'm a LITTLE wary of using seconds and not minutes or even hours as the duration. I don't think we need to be exact down to the second, and humans have a really hard time with big numbers. It's gonna be pretty easy to mess up and have a duration that's much too short. It helps that you have a comment with an example, but remember that in many cases we'll be setting this value in puppet -- i.e., without this helpful hint.